src/logger.ts

```diff
@@ -1,6 +1,72 @@
-import * as winston from 'winston'
+import chalk from 'chalk'
+import { join } from 'path'
+import { createLogger, format, transports } from 'winston'
+import * as Transport from 'winston-transport'
+import { configDir } from './conf'
 
-// TODO: Configure transport to send errors to Splunk
-winston.cli()
+// Setup logging
+const VERBOSE = '--verbose'
+const isVerbose = process.argv.indexOf(VERBOSE) >= 0
 
-export default winston
+const { combine, timestamp, colorize } = format
+
+export const DEBUG_LOG_FILE_PATH = join(configDir, 'vtex_debug.txt')
```

Comment:
*nit*: can you put a comment above this line with the complete path of this file?